docs/design/numeric_literals.md

```diff
@@ -1,37 +1,44 @@
-# Numeric Literals
+# Numeric Literal Semantics
 
 <!--
 Part of the Carbon Language project, under the Apache License v2.0 with LLVM
 Exceptions. See /LICENSE for license information.
 SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 -->
 
+> **STATUS:** Up-to-date on 23-Aug-2022.
+
 <!-- toc -->
 
 ## Table of contents
 
+-   [TODO](#todo)
 -   [Overview](#overview)
     -   [Defined Types](#defined-types)
-    -   [Integers](#integers)
-    -   [Floats](#floats)
+    -   [Numeric literal syntax](#numeric-literal-syntax)
     -   [Implicit conversions](#implicit-conversions)
     -   [Examples](#examples)
--   [Numeric literal semantics](#numeric-literal-semantics)
--   [Open questions](#open-questions)
-    -   [Primitive types as code vs built-in](#primitive-types-as-code-vs-built-in)
-    -   [Syntax for wrapping operations](#syntax-for-wrapping-operations)
-    -   [Non-power-of-two sizes](#non-power-of-two-sizes)
+-   [Alternatives Considered](#alternatives-considered)
 -   [References](#references)
 
 <!-- tocstop -->
 
+## TODO
```

Comment:
I don't think this should be up front, the overview is much better content to start off with. This should just be a note on the relevant section.